gapic/src/main/com/google/gapid/perfetto/models/SliceTrack.java

```diff
@@ -486,4 +488,17 @@ public Node build(Map<Long, List<Builder>> byParent) {
       }
     }
   }
+
+  public static RGBA getColor(String title, int depth) {
+    return colorForSlice(title, depth, 0);
+  }
+
+  public static RGBA getBorderColor(String title, int depth) {
+    return colorForSlice(title, depth, StyleConstants.isLight()? -5 : 5);
+  }
+
+  private static RGBA colorForSlice(String title, int depth, int shadeIdx) {
+    return StyleConstants.Palette.getColor(
+        (title.hashCode() ^ Integer.hashCode(depth)) & 0x7fffffff, shadeIdx);
```

Comment:
`Integer.hashCode` is the identity function (does nothing).

Reply:
Interesting to learn, thanks!